irma/cmd/session.go

```diff
@@ -42,42 +44,69 @@ with --message flags, or it can be specified as JSON to the --request flag.`,
 irma session --sign irma-demo.MijnOverheid.root.BSN --message message
 irma session --issue irma-demo.MijnOverheid.ageLower=yes,yes,yes,no --disclose irma-demo.MijnOverheid.root.BSN
 irma session --request '{"type":"disclosing","content":[{"label":"BSN","attributes":["irma-demo.MijnOverheid.root.BSN"]}]}'
-irma session --server http://localhost:8088 --authmethod token --key mytoken --disclose irma-demo.MijnOverheid.root.BSN`,
+irma session --server http://localhost:8088 --authmethod token --key mytoken --disclose irma-demo.MijnOverheid.root.BSN
+irma session --url http://localhost:8088 --static mystaticsession
```

Comment:
I use the --url option now, whereas normally the --server option would be logical here. But a user should enter the external url to make sure the IRMA app can reach it. This means the `--url` parameter description better matches, but this `--url` option is normally only used when running in library mode (so when not talking to another irma server instance).

It's both a bit ugly. What would you prefer @synaptic-cleft consistency in explanation, or consistency in which option is used when?

Reply:
Or I should make a dummy call to the`--server` to check which external url is configured, such that we can use that one in the static QR. That would be best, but that is also a bit weird.

Reply:
I probably would have used server here, yeah, but I don't have a clear preference. I don't mind if it stays how you currently implemented it.

Maybe @sietseringers has a more pronounced opinion here.

Reply:
I agree that neither `--server` nor `--url` is a perfect match here. However, those two roughly correspond to the current two modes `irma session` supports (external server vs internal library), while the new static functionality is different from both of those.

Would it not be better to choose neither one, and introduce a new flag, e.g. `--static-server`? This allows us to write a clear parameter description for that new parameter, and it means that the `irma server` command will have three distinct options (`--url`, `--server`, `--static-server`) for each of its three modes (library, external server, static sessions).

Reply:
Maybe that's the best solution indeed. A bit more verbose but then we can get the explanations clear. I'll update that.